packages/shared/src/contexts/UITaskManager.tsx

```diff
@@ -68,16 +64,16 @@ export const createUITaskManager = <
                         Component,
                         {
                             key: task.id,
-                            open: true,
                             ...task.options,
+                            open: true,
```

Comment:
I think we should move `key: task.id` after `...task.options` as well.
```suggestion
                            open: true,
                            key: task.id,
```